src/main/kotlin/com/fortysevendeg/hood/github/GithubCommitIntegration.kt

```diff
@@ -0,0 +1,96 @@
+package com.fortysevendeg.hood.github
+
+import arrow.core.None
+import arrow.core.Option
+import arrow.core.some
+import arrow.effects.IO
+import com.fortysevendeg.hood.GhCreateCommit
+import com.fortysevendeg.hood.GhFileSha
+import com.fortysevendeg.hood.GhInfo
+import com.fortysevendeg.hood.GhUpdateCommit
+import org.gradle.api.GradleException
+import org.http4k.client.DualSyncAsyncHttpHandler
+import org.http4k.client.OkHttp
+import org.http4k.core.*
+import org.http4k.format.Jackson
+import org.http4k.format.Jackson.auto
+import org.http4k.format.Jackson.json
+
+object GithubCommitIntegration {
+
+  private val client: DualSyncAsyncHttpHandler = OkHttp()
+
+  private fun buildRequest(method: Method, info: GhInfo, url: String): Request {
```

Comment:
💡 The requests that are built for commits looks like the same than comments. If the configuration is expected to always be the same maybe it can be delegated to a common class

https://github.com/47deg/hood/blob/28fa771b4c5a32ae19ef7a62c04e4721622997e7/src/main/kotlin/com/fortysevendeg/hood/github/GithubCommentIntegration.kt#L21-L32